examples/model_selection/plot_roc.py

```diff
@@ -437,7 +439,19 @@
 # global performance of a classifier can still be summarized via a given
 # averaging strategy.
 #
-# Micro-averaged OvR ROC is dominated by the more frequent class, since the
-# counts are pooled. The macro-averaged alternative better reflects the
-# statistics of the less frequent classes, and then is more appropriate when
-# performance on all the classes is deemed equally important.
+# When dealing with imbalanced datasets, choosing the correct metric is crucial.
+# Accuracy may not always reflect model performance well in these cases.
+# For example, metrics such as F1 score, precision, or recall might be more
+# appropriate, depending on the business context or problem you are addressing.
+# It is also essential to select an appropriate averaging method (micro vs. macro)
+# depending on the problem at hand:
+#
+# - Micro-averaging aggregates metrics across all instances, treating each
+#   individual instance equally, regardless of its class. This approach is useful
+#   when evaluating overall performance, but note that it can be dominated by
+#   the majority class in imbalanced datasets.
+#
+# - Macro-averaging calculates metrics for each class independently and then
+#   averages them, giving equal weight to each class. This is particularly useful
+#   for imbalanced datasets, as it ensures that performance on under-represented
+#   classes is given the same importance as the more frequent classes.
```

Comment:
This implies that this is better for imbalanced datasets, whereas I think a better way to phrase it could be something like:

```suggestion
#   averages them, giving equal weight to each class. This is particularly useful
#   when you want under-represented to be considered as important as highly
#   populated classes.
```

as described here: https://github.com/scikit-learn/scikit-learn/issues/28585#issuecomment-1989484965

Reply:
@lucyleeow, thanks for your review, both statements essentially mean the same thing, it's just phrasing, we can wait for the maintainers to review this PR and if required I can make the required change.

Reply:
Hi @lucyleeow, Very sorry about before, I am new to contributing to sklearn and wasn't aware you were a maintainer for the documentation team, I have implemented the changes you suggested. RFR. Thank you!